concepts/conditionals/about.md

```diff
@@ -0,0 +1,178 @@
+# About
+
+## Conditional Statements
+
+Conditional statements are decision-making statements that can either execute
+side-effects or control the flow of execution through a program. Conditional 
+statements include `if`, `if/else`, `switch`, and `ternary` statements.
+
+Conditional statements are used to ask questions about a value or a relation of
+values. The question evaluates to a Boolean answer, but the `if` statement
+itself can return any value or it can simply execute a side-effect.
+
+## If Statements
+
+The simplest `if` statement evaluates an expression and executes one line. The
+expression being tested is always included in parentheses.
+
+It can execute a side-effect
+
+```c
+if (count > 10)
+    printf("count is greater than 10");
+```
+
+or it can change the flow of execution
+
+
+```c
+if (count > 10)
+    return sum;
+```
+
+If an `if` statement executes more than one line it must place the lines within
+curly brackets
+
+```c
+if (count > 10) {
+    printf("count is greater than 10");
+    return sum;
+}
+```
+
+Some coding teams may standardize that `if` statements should always use curly
+braces, even for one line.
+
+
+## If Else
+
+An `if` statement can be coupled with an `else`
+
+```c
+if (number % 2 == 0)
+    return "even";
+else
+    return "odd";
+```
+
+## If Else If
+
+
+`else` can be coupled with another `if` to create what's called
+an `if-else-if ladder`.
+
+```c
+if (number == 0)
+    return "zero";
+else if (number < 0)
+    return "negative";
+else if (number < 10)
+    return "single digit";   
+else if (number % 2 == 0)
+    return "even";
+else
+    return "odd";
+```
+
+## Switch
+
+
+Sometimes an `if-else-if ladder` can be better represented by a
+`switch`. Like `if`, the expression being tested is always put
+in parentheses.
+
+```c
+switch (number) {
+    case -1: return "negative one";
```

Comment:
Would be good to put the case bodies (i.e. the `return foo;`)   etc on a new line in their own block, as this is the style we use in the exercises.
I think it helps to make it clearer what is part of the case and what the case matches versus what the the case does

Reply:
Same is true for the other switch cases.